sdcm/sct_provision/__init__.py

```diff
@@ -10,3 +10,8 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2021 ScyllaDB
+
+from sdcm.sct_provision.azure.azure_instance_request_definition_builder import azure_instance_request_builder
+from sdcm.sct_provision.instances_request_definition_builder import instances_request_builder
+
+instances_request_builder.register_builder(backend="azure", builder_function=azure_instance_request_builder)
```

Comment:
why is it `azure` the default in here?
isn't this `__init__.py` class common to all backends?

Reply:
it's about registering builder for azure. We need to create and register builders for other backends as well.